src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -69,6 +71,7 @@
 public class CuDisbursementVoucherDocument extends DisbursementVoucherDocument implements CULegacyTravelIntegrationInterface {
 
     private static final long serialVersionUID = 1L;
+    protected static Logger LOG = LogManager.getLogger();
```

Comment:
This should be declared as a private static final variable.

Reply:
updated